torch/amp/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .grad_scaling import scale_outputs, \
```

Comment:
i'd think this should be `torch.cuda.amp` and not `torch.amp`, to begin with.

Reply:
The current PR effectively  mandates that change. There are many CUDA-specific checks.

Reply:
That seems fine to me.